client/src/app/_models/Barman.ts

```diff
@@ -28,6 +28,10 @@ export class Barman {
     roles?: AssociationChanges,
   };
 
+  public isActive(): Boolean {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this 'Boolean' wrapper object with primitive type 'boolean'. ([typescript:S1533](https://sonarcloud.io/organizations/k-fet/rules#rule_key=typescript%3AS1533))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=k-app&pullRequest=302&issues=AWdH7aJv1-jKZJbrex9j&open=AWdH7aJv1-jKZJbrex9j)

[ISSUE_KEY]: AWdH7aJv1-jKZJbrex9j